src/Models/HydrostaticFreeSurfaceModels/hydrostatic_prescribed_velocity_fields.jl

```diff
@@ -0,0 +1,81 @@
+#####
+##### PrescribedVelocityFields
+#####
+
+using Oceananigans.Grids: Center, Face
+using Oceananigans.Fields: FunctionField
+
+
+using Oceananigans.BoundaryConditions: fill_halo_regions!
+using Oceananigans.TimeSteppers: ab2_step_field! 
```

Comment:
Does this mean you can't use `rk3`?  Not a problem, just curious.  If it's using the implicit `ab2` then I guess that would mean no `rk3`.

Reply:
That's right --- `HydrostaticFreeSurfaceModel` only works with AB2 right now.